src/sirius/types.h

```diff
@@ -80,6 +80,21 @@ struct Size {
     int col{0};
 };
 
+struct Point {
+    Point() = default;
+
+    Point(int x, int y) : x(x), y(y) {}
+
+    ~Point() = default;
```

Comment:
Defining all the 5 operations seems overkill.

Beside I'd have named it `coordinates`. And it be nice to have arithmetic with another _delta_ type.

```
coord + delta -> coord
delta + coord -> coord
coord - coord -> delta
delta + delta -> delta
delta - delta -> delta
factor * delta -> delta
delta * factor -> delta
delta / factor -> delta
``